XMLtoMD/GuidelineXmlToMD/Program.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text.RegularExpressions;
+using MarkdownOut;
+
+
+namespace GuidelineXmlToMD
+{
+    class Program
+    {
+        static MarkdownOut.MdWriter _MdWriter;
+        static void Main(string[] args)
```

Comment:
You might consider leveraging [System.CommandLine](https://github.com/dotnet/command-line-api) (specifically DragonFruit) to make this easier.